examples/indigo_job_output_upload_web.yaml

```diff
@@ -0,0 +1,141 @@
+
+description: >
+  TOSCA examples for specifying a Chronos Job that runs an application using the input stored at some URL and uploads the output data to an http(s) or ftp(s) or webdav(s) repository
+
+inputs:
+
+   input_urls:
+     type: list
+     description: List of input files that will be downloaded in the job sandbox (archives will be automatically uncompressed)
+     entry_schema:
+       type: string
+     required: yes
+ 
+   output_protocol:
+     type: string
+     description: Protocol that will be used to upload the output generated by the job
+     constraints:
+        - valid_values: [ http, https, webdav, webdavs, ftp, ftps ]
+     required: yes
+
+   output_endpoint:
+     type: string
+     description: Endpoint to be used to upload the output generated by the job. Example: http://myhttpserver:8080/
+     required: yes
+
+   output_path:
+     type: string
+     description: Path in the repository where the output data will be uploaded (to be appended to the endpoint string, e.g. http://myhttpserver:8080/output_path/)
+     required: yes
+
+   output_username:
+     type: string
+     description: Username used to access the URL where the output files will be stored
+     required: yes
+
+   output_password:
+     type: string
+     description: Protocol that will be used to upload the output generated by the job
+     required: yes
+
+   output_filenames:
+     type: list
+     description: List of filenames generated by the application run
+     entry_schema:
+       type: string
+     required: yes
+
+   command:
+     type: string
+     description: Command to execute
+     default: "/bin/bash run.sh"
+     required: no
+
+   cpus:
+     type: float
+     description: Amount of CPUs for this job
+     required: yes
+   
+   mem:
+     type: float
+     description: Amount of Memory (MB) for this job
+     required: yes
+
+   docker_image:
+     type: string
+     description: Docker image to be used to run the container application
+     required: yes
+     
+topology_template:
+
+  node_templates:
+
+    chronos_job:
+      type: tosca.nodes.indigo.Container.Application.Docker.Chronos
+      properties:
+        schedule: 'R0/2015-12-25T17:22:00Z/PT1M'
+        description: 'Execute Application'
+        command: { get_input: command }
+        uris: { get_input: input_urls}
+        retries: 3
+        environment_variables:
+          OUTPUT_FILENAMES: { get_input: output_filenames }
+          ONEDATA_SERVICE_TOKEN: "TOKEN_TO_BE_SET_BY_THE_ORCHESTRATOR"
+          ONEDATA_SPACE: "DATA_SPACE_TO_BE_SET_BY_THE_ORCHESTRATOR"
+          ONEDATA_PATH: "PATH_TO_BE_SET_BY_THE_ORCHESTRATOR"
+          ONEDATA_PROVIDERS: "ONEDATA_PROVIDERS_TO_BE_SET_BY_THE_ORCHESTRATOR"
+      artifacts:
+        image:
+          file: { get_input: docker_image }
+          type: tosca.artifacts.Deployment.Image.Container.Docker
+      requirements:
+        - host: docker_runtime1
+
+
+  chronos_job_upload:
```

Comment:
@maricaantonacci, you  have to add two spaces before the chronos_job_upload node to have the correct alignement.